lib/imapSimple.js

```diff
@@ -384,6 +443,19 @@ function connect(options, callback) {
         imap.once('error', imapOnError);
         imap.once('close', imapOnClose);
         imap.once('end', imapOnEnd);
+
+        if (options.hasOwnProperty('onmail')) {
+            imap.on('mail', options.onmail);
+        }
+
+        if (options.hasOwnProperty('onexpunge')) {
+            imap.on('expunge', options.onexpunge);
+        }
+
+        if (options.hasOwnProperty('onupdate')) {
+            imap.on('update', options.onupdate);
+        }
```

Comment:
Do you mind adding documentation to describe when these events would be useful and the options to add listeners for them in the README?